lib/usbmux/index.js

```diff
@@ -61,13 +62,28 @@ function swap16 (val) {
  * @throws {B.TimeoutError} if connection timeout happened
  * @returns {Promise<NodeJS.Socket>} Connected socket instance
  */
-async function getDefaultSocket (opts = {}) {
-  const {
-    socketPath = DEFAULT_USBMUXD_SOCKET,
-    socketPort = DEFAULT_USBMUXD_PORT,
-    socketHost = DEFAULT_USBMUXD_HOST,
-    timeout = 5000,
-  } = opts;
+async function getDefaultSocket(opts = {}) {
+  const defaults = {
+    socketPath: DEFAULT_USBMUXD_SOCKET,
+    socketPort: DEFAULT_USBMUXD_PORT,
+    socketHost: DEFAULT_USBMUXD_HOST,
+    timeout: 5000
+  };
+
+  if (process.env.USBMUXD_SOCKET_ADDRESS) {
+    log.debug(`Using USBMUXD_SOCKET_ADDRESS environment variable as default socket: ${process.env.USBMUXD_SOCKET_ADDRESS}`);
```

Comment:
this log is confusing as it would be printed anyway even if actual opts are provided.
Consider checking if any option is already provided in the condition above instead

Reply:
Updated